test/test_dynamicLeaderboard.py

```diff
@@ -0,0 +1,26 @@
+from testing_base import *
+
+def test_dynamicLeaderboard():
+
+    donovan = Player('Donovan')
+    jesse = Player('Jesse')
+    roy = Player('Roy')
+    jonny = Player('Jonny')
+
+    players = [donovan, jesse, roy, jonny]
+
+    # Add cards to players' hand
+    for p in players: 
+        for i in range(6):
+            p.addCard(Card("Hearts", random.randint(1, 10), [], []))
+
+    for p in players:
+        print(p.showValue(), p.name)
+    
+    winner = showWinner(players)
+
+    print(f'\nWinner: {winner.name}!')
+    updateLeaderboard(winner.name)
+    showLeaderboard()
+
```

Comment:
This testing is well done. My only thought is what we might want to do with ties. If two people have the same hand, who wins?

However I'm not really how we would want to handle ties from a gameplay perspective anyway. This can be another thing to think about for next sprint.

Reply:
Good point. We can try to incorporate that this sprint.